shortcodes/sc-icon.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Provides a means to insert an empty span tag with icon classes.
+ **/
+if ( ! class_exists( 'IconSC' ) ) {
+	class IconSC extends ATHENA_SC_Shortcode {
+		public
+			$command = 'icon',
+			$name = 'Icon',
+			$desc = 'Adds a span, classes and styles intended for icons.',
+			$group = 'Athena Framework - Utilities';
+
+		/**
+		 * Returns the shortcode's fields.
+		 *
+		 * @author RJ Bruneel
+		 * @since 0.2.0
+		 *
+		 * @return Array | The shortcode's fields.
+		 **/
+		public function fields() {
+			return array(
+				array(
+					'param'   => 'class',
+					'name'    => 'CSS Classes',
+					'desc'    => 'Include icon classes to be added to the span.',
+					'type'    => 'text'
+				),
+				array(
+					'param'   => 'style',
+					'name'    => 'Inline Styles',
+					'desc'    => 'Any additional styles to be added to the span.',
+					'type'    => 'text'
+				)
+			);
+		}
+
+		/**
+		 * Adds an empty span with supplied classes and styles along with aria-hidden="true" attribute.
+		 **/
+		public function callback( $atts, $content='' ) {
+			$atts = shortcode_atts( $this->defaults(), $atts );
+
+			$classes = $atts['class'] ?: false;
```

Comment:
Since a default isn't being set, this should probably be a `isset()` check to avoid a unset key warning. I'm not 100% sure if the warning will get thrown or not, but using `isset` will avoid it for sure. Same for the next line.

Reply:
`$atts['class']` will always be set in `$this->defaults()`'s returned value, so no `isset()` check is needed :)
https://github.com/UCF/Athena-Shortcodes-Plugin/blob/master/includes/class-shortcode.php#L55-L72